doc/overview.md

```diff
@@ -0,0 +1,149 @@
+
+### Step 3: Get a set of sample credentials
+
+Go to the **Applications** tab, click on any application using the service you're configuring, and get the **API credentials**. If you do not have applications/ users yet, you can create an application yourself (from the details page of any individual developer account), the credentials will be generated automatically.
+
+Typically the credentials will be a ```user_key``` or the pair ```app_id/app_key``` depending on the authentication mode you are using (note that the 3scale hosted gateway does not currently support OAuth, however you can configure it if you are using the self-managed gateway integration.
+
+### Step 4: Get a working request
+
+We are almost ready to roll. Go to your browser (or command-line curl) and do a request to your own API to check that everything is working on your end.
+
+For instance it could be something like this:
+
+```
+http://api-sentiment.3scale.net/v1/word/good.json
+```
+Note that you are not using 3scale's gateway yet. You are just getting a working example that will be used in the next step.
+
+### Step 5: Get a working request to your API
+
+Now do the same request but replacing your **private base URL** (in the example ```http://api-sentiment.3scale.net:80```) by your hosted endpoint (e.g. if you were Twitter, you would need to change the ```https://api.twitter.com``` to ```https://api-xxxxxxxxxxxxx.staging.apicast.io```). You also have to add the parameters to pass the credentials that you just copied.
+
+Continuing the example in this tutorial it would be something like:
+
+```
+https://api-2445581436380.staging.apicast.io:443/v1/word/good.json?user_key=YOUR_USER_KEY
+```
+
+If you execute this request you will get the same result as in Step 4. However, this time the request will go through the 3scale hosted gateway.
+
+**And that's it! You have your API integrated with 3scale**.
+
+3scale's hosted gateway does the validation of the credentials and applies any gateway rules that you have defined to handle rate-limits, quotas and analytics. If you did not touch the mapping rules every request to the gateway will increase the metric ```hits``` by 1, you can check in your admin console how the metric ```hits``` is increased.
+
+If you want to experiment further, you can test what happens if you try credentials that do not exist. The gateway will respond with a generic error message (you can define your custom one).
+
+You can also define a rate limit of 1 request per minute. After you try your second request within the same minute you will see that the request never reaches your API backend. The gateway stops the request because it violates the quota that you have just set up.
+
+## Integration configuration

```

Comment:
I have a general comment that this is much more than an overview of apicast - it's a tutorial on API Integration (using apicast)..... all the parts above I reviewed I am fine with..... but maybe these points below are going too far out of scope and are really documentation of the Admin Portal?